.travis.yml

```diff
@@ -27,6 +27,9 @@ env:
     # (Useful for builds which don't need to repeat code checks)
     - SKIP_CODE_CHECKS="-Denforcer.skip=true -Dcheckstyle.skip=true -Dlicense.skip=true -Dxml.skip=true"
 
+before_install:
+  # Remove outdated settings.xml from Travis builds. Workaround for https://github.com/travis-ci/travis-ci/issues/4629
+  - rm ~/.m2/settings.xml
```

Comment:
This looks to be an accidental change?  It was removed a while back from our Travis settings as it's no longer needed.